src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,21 @@
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    completed, id, title, user,
+  } = todo;
```

Comment:
It is better to provide a new line for each prop if their amount is > 3
```suggestion
  const {
    completed, 
    id, 
    title, 
    user,
  } = todo;
```